cbor_merge/MarketTypes.sol

```diff
@@ -0,0 +1,127 @@
+/*******************************************************************************
+ *   (c) 2022 Zondax AG
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ********************************************************************************/
+//
+// DRAFT!! THIS CODE HAS NOT BEEN AUDITED - USE ONLY FOR PROTOTYPING
+
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.17;
+
+import "../cbor/BigIntCbor.sol";
+import "./CommonTypes.sol";
+
+/// @title Filecoin market actor types for Solidity.
+/// @author Zondax AG
+library MarketTypes {
+    CommonTypes.FilActorId constant ActorID = CommonTypes.FilActorId.wrap(5);
+    uint constant AddBalanceMethodNum = 822473126;
+    uint constant WithdrawBalanceMethodNum = 2280458852;
+    uint constant GetBalanceMethodNum = 726108461;
+    uint constant GetDealDataCommitmentMethodNum = 1157985802;
+    uint constant GetDealClientMethodNum = 128053329;
+    uint constant GetDealProviderMethodNum = 935081690;
+    uint constant GetDealLabelMethodNum = 46363526;
+    uint constant GetDealTermMethodNum = 163777312;
+    uint constant GetDealEpochPriceMethodNum = 4287162428;
+    uint constant GetDealClientCollateralMethodNum = 200567895;
+    uint constant GetDealProviderCollateralMethodNum = 2986712137;
+    uint constant GetDealVerifiedMethodNum = 2627389465;
+    uint constant GetDealActivationMethodNum = 2567238399;
+    uint constant PublishStorageDealsMethodNum = 2236929350;
```

Comment:
I don't understand why we're copying over the entire file. We can just have our custom types in a single file on our end. I will refactor it that way.